mqtt/mqtt-bridge/src/client.rs

```diff
@@ -89,27 +195,37 @@ pub struct MqttClient<T>
 where
```

Comment:
Can you please put some comments on the purpose of this client? It's easy to get lost b/w all the clients that we already have, so let's keep it documented :)